test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_ListDescribeConsumerGroups.cs

```diff
@@ -63,16 +63,68 @@ private void checkConsumerGroupDescription(
         [Theory, MemberData(nameof(KafkaParameters))]
         public void AdminClient_ListDescribeConsumerGroups(string bootstrapServers)
         {
+            var groupID = Guid.NewGuid().ToString();
+            var nonExistentGroupID = Guid.NewGuid().ToString();
```

Comment:
Renamed to `groupId` and `nonExistentGroupId`

Reply:
ok